src/oss/confluent.sh

```diff
@@ -1521,9 +1529,17 @@ consume_command() {
       export_kafka
       bootstrapserver="--bootstrap-server localhost:${kafka_port}"
     fi
-    removestr="--value-format avro"
-    if [[ "$args" =~ "$removestr" ]]; then
-      args=${args//$removestr/}
+    cloud="--cloud"
+    if [[ "$args" =~ "$cloud" ]]; then
```

Comment:
basically this is the same code as above, just with a different param. Ideally we would factor out in a function and call it in both places with producer or consumer arguments. See `extract_name_from_properties_file` and how it's used for a very similar example of such a function.

Reply:
Great suggestion!